crates/bitwarden/src/auth/api/response/identity_token_response.rs

```diff
@@ -14,30 +14,30 @@ pub enum IdentityTokenResponse {
     Authenticated(IdentityTokenSuccessResponse),
     Payload(IdentityTokenPayloadResponse),
     Refreshed(IdentityTokenRefreshResponse),
-    TwoFactorRequired(IdentityTwoFactorResponse),
+    TwoFactorRequired(Box<IdentityTwoFactorResponse>),
```

Comment:
Why did we need a box here?

Reply:
`IdentityTwoFactorResponse` is too big and so it bloats the size of the `IdentityTokenResponse` enum, as it's size is based on the biggest element it contains.

The size in bytes of the elements:
```rust
        assert_eq!(240, std::mem::size_of::<IdentityTokenSuccessResponse>());
        assert_eq!(128, std::mem::size_of::<IdentityTokenPayloadResponse>());
        assert_eq!(104, std::mem::size_of::<IdentityTokenRefreshResponse>());
        assert_eq!(656, std::mem::size_of::<IdentityTwoFactorResponse>());
        assert_eq!(120, std::mem::size_of::<IdentityCaptchaResponse>());
        assert_eq!(96, std::mem::size_of::<IdentityTokenFailResponse>());
```